guides/app-guides/agEUR/borrow.md

```diff
@@ -2,58 +2,95 @@
 description: How to borrow agEUR from the Angle App
 ---
 
-# 🏦 Borrowing agEUR
+# 🏦 Borrow
 
 The [Angle Borrowing module](/borrowing-module/README.md) lets you borrow agEUR from crypto collateral **on different chains**.
 
-Essentially, you can deposit your crypto into a vault, to get a stablecoins loan in exchange. If the value of the crypto you deposited goes below a certain amount compared to the value of your loan, you can get [liquidated](/borrowing-module/vaults/liquidations.md).
+Essentially, you open a vault with a specific token as collateral, and can get a stablecoin loan in exchange. If the value of the vault collateral goes below a certain amount compared to the value of your loan, you can get [liquidated](/borrowing-module/vaults/liquidations.md).
 
-When borrowing agEUR, you can directly swap the borrowed agEUR into more of the token originally used as collateral to get leverage. Learn more about the mechanism [here](/borrowing-module/vaults/README.md#leveraging-collateral-exposure).
+When opening a vault, you can deposit any token and it will be automatically swapped into the selected vault collateral. When borrowing agEUR, you can directly swap the borrowed agEUR into more of the token originally used as collateral to get leverage. All of this is done in the same transaction.
```

Comment:
Let's keep this paragraph as before -> we should do the difference between protocol description (like in here and guides description): it will automatically be swapped is not a feature of the protocol, it's a feature of the app